lib/signaling/v2/room.js

```diff
@@ -184,7 +213,26 @@ RoomV2.prototype._update = function _update(roomState) {
  * @returns {Promise.<Array<StatsReport>>}
  */
 RoomV2.prototype.getStats = function getStats() {
-  return this._peerConnectionManager.getStats();
+  var self = this;
+
+  function filterLocalTrackStats(trackStats) {
+    return self._published.has(trackStats.trackId);
```

Comment:
@markandrus Can't we get this from `[...this.localParticipant.tracks.values()].filter(track => !!track.sid)`?